.github/workflows/infra.yml

```diff
@@ -0,0 +1,108 @@
+# This workflow will deploy and configure clusters necessary to run integration tests
+#
+# To configure this workflow:
+#
+# 2. Setup secrets in your repository: IBM_CLOUD_API_KEY
+# 3. Change the values for the IC_REGION
+
+name: Deploy and Configure tests clusters
+
+on:
+  push:
+    paths:
+    - 'scripts/ansible/**'
+    - '.github/workflows/infra.yml'
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  IC_API_KEY: ${{ secrets.IBM_CLOUD_API_KEY }}
+  IC_REGION: eu-de
+
+jobs:
+  deploy-config-openshift-cluster:
+    name: Deploy and Configure OpenShift cluster
+    runs-on: ubuntu-latest
```

Comment:
Can we replace:
    `runs-on: ubuntu-latest`
with:
    runs-on: ${{matrix.os}}
    strategy:
        os: [ubuntu-latest, windows-latest, macOS-latest]